src/file.js

```diff
@@ -88,13 +88,19 @@ exports.write = function (fileEntry, data, options = {}) {
 exports.read = function (fileEntry, options = {}) {
   return utils.promiseCall(fileEntry, 'file')
     .then(file => {
-      return new Promise((resolve, reject) => {
-        const reader = new FileReader();
-        reader.onload = () => resolve(reader.result);
-        reader.onerror = () => reject(reader.error);
-        // see: https://developer.mozilla.org/ru/docs/Web/API/FileReader
-        readAs(options.type, reader, file);
-      });
+      if (options.type === 'Blob') {
+        return new Response(file).blob();
```

Comment:
Could we achieve that without using `Response`?

Reply:
Technically `File` is a `Blob`.   This could just be the same as `MutableFile`

Reply:
> Could we achieve that without using `Response`?

I tried different APIs, but they either require reading the entire Blob into a buffer, or do not (deep) clone the blob at all. `Response` is so far the best I can find.

Reply:
Ok, I'm for simplicity, using `Response` is now clear. 
I just suggest to make a util function `fileEntryToBlob` and put your comment why we using Response here.